mappings/net/minecraft/class_2152.mapping

```diff
@@ -1,4 +0,0 @@
-CLASS net/minecraft/class_2152
```

Comment:
Can you revert this? sky changes stuff in this class in his pr too.

Reply:
You know what, will do that myself.

Reply:
ask sky
i did it first :hahayes:

Reply:
His pr is more huge, so doing it here is better